lang-portal/backend-go/internal/database/seed.go

```
@@ -33,19 +33,26 @@ func SeedDatabase(db *sql.DB) error {
 	}
 
 	// Seed words
-	words, err := seedWords(tx)
+	wordIDs, err := seedWords(tx)
 	if err != nil {
 		tx.Rollback()
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)